src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml.cs

```diff
@@ -458,7 +458,6 @@ private void mniShowAncestry_Click(object sender, RoutedEventArgs e)
         private void mniShowAncestry_Loaded(object sender, RoutedEventArgs e)
         {
             ((MenuItem)sender).IsChecked = Configuration.ShowAncestry;
-            SetFocusOnHierarchyTree();
```

Comment:
This was setting the focus on menu load, but it didn't actually manifest until after the menu went away